ibis/backends/pandas/execution/util.py

```diff
@@ -75,9 +75,12 @@ def compute_sorted_frame(df, order_by, group_by=(), timecontext=None, **kwargs):
             new_columns[computed_sort_key] = temporary_column
 
     result = df.assign(**new_columns)
-    result = result.sort_values(
-        computed_sort_keys, ascending=ascending, kind="mergesort"
-    )
+    try:
+        result = result.sort_values(
+            computed_sort_keys, ascending=ascending, kind="mergesort"
+        )
+    except TypeError:
+        result = result.sort_values(computed_sort_keys, ascending=ascending)
```

Comment:
This code is hit by both pandas and dask, so I kept the original implementation around.